test/src/edu/stanford/nlp/dcoref/RulesTest.java

```diff
@@ -20,12 +19,12 @@
   List<CoreLabel> MIBM = SentenceUtils.toCoreLabelList("MIBM");
 
   public void testIsAcronym() {
-    assertTrue(Rules.isAcronym(IBM, IBM2));
-    assertTrue(Rules.isAcronym(IBM2, IBM));
-    assertFalse(Rules.isAcronym(IBM, IBMM));
-    assertFalse(Rules.isAcronym(IBM2, IBMM));
-    assertFalse(Rules.isAcronym(IBM, MIBM));
-    assertFalse(Rules.isAcronym(IBM2, MIBM));
+    assertTrue("Acronym IMB -> International Business Machines",Rules.isAcronym(IBM, IBM2));
```

Comment:
You get a line number for where the test fails so I don't really agree this is an mystery assertion.  If you want to gather up a bunch of these changes into a single change, though, I can merge it